windows/ReactNativeWebView2/PropertySheet.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Label="PropertySheets" />
+  <PropertyGroup Label="UserMacros" />
+  <!--
+    To customize common C++/WinRT project properties: 
```

Comment:
why do we need 2 solutions/projects? I thought the idea was a single project would target both? if not, does the app always us the reactnativewebview2 project?

Reply:
Currently we chose this design: 
"Toggling Between the WebView and WebView2 Control
The toggle from WebView to WebView2 will be handled from the JavaScript. In this solution, a Windows specific prop would be added to the WebView component called useWebView2. **Next, a separate native module for WebView2 based off of the original WebView code is created. Now we have two windows projects. Then both modules are imported to the JavaScript and, based on the value of the useWebView2 prop, which defaults to false if not specified, then the ReactNativeWebView or ReactNativeWebView2 module is returned as the WebView component**. With this solution, developers can toggle between WebView and WebView2 at runtime by adjusting the value of the useWebView2 prop. Note this solution matches the design of the react-native-datetimepicker community module which provides both the TimePicker and DateTimePicker components.

The app will use the reactnativewebview project when the app is using `<WebView />` and will use the reactnativewebview2 project when using `<WebView useWebView2={true} />`
![image](https://user-images.githubusercontent.com/34109996/151080877-cdaa340b-82a9-4fbc-944c-89e751b15458.png)


I should be able to take the two modules and house them in once vcxproj if we'd prefer to only deal with one project file. We could also save this work as feature work for the future if the two project structure gives people issues.